subprojects/docs/src/samples/userguide/plugins/readOnlyNestedProperty/groovy/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'serviceInjection'
```

Comment:
I think this should say `readOnlyNestedProperty` or somesuch, no?

Reply:
Fixed.